packages/react/src/components/Text/Text.styles.ts

```diff
@@ -41,7 +41,7 @@ export const H3 = {
 
 export const H4 = {
   ...base,
-  fontSize: 'md',
+  fontSize: 'lg',
```

Comment:
More of an aesthetic opinion here, but `lg` puts H4 only 2pts away from H3 and makes them less visually distinct.